src/utils.rs

```diff
@@ -0,0 +1,54 @@
+use macroquad::{
```

Comment:
while this is used just by one weapon - I would consider moving it to this weapon and generalize later

Reply:
It's about to be used for the mines (being worked on by @kindaro (it's here now!)), so is it fine if I leave it as is? I'll also probably use it for the kickbombs or something like that next.

Reply:
I would prefer to copy-paste this for each weapon (1 or 2 its not a big deal), but up to you, I can live with utils.rs

Reply:
If the explosive class is used for all explosive weapons, the util class should no longer be necessary. I'll move it